source/changelogs/2022-02-01-February.md

```diff
@@ -0,0 +1,30 @@
+---
+title: February 2022 Changelog
+changelog: true
+description: February 2022 Pantheon changelog
+reviewed: "2022-02-01"
+---
+
+## Platform Improvements
+
+### Drupal 9.3.3
+
+[Drupal 9.3.3](https://www.drupal.org/project/drupal/releases/9.3.3) is available for production sites and includes backwards-compatible improvements. [Drupal 9.3](https://www.drupal.org/project/drupal/releases/9.3.0) was released in December 2021. From Drupal 9.3, PHP version 8.0 or higher is recommended; PHP 7.3 and higher remains supported. [Drupal 9](/drupal-9) sites on Pantheon use [Integrated Composer](/integrated-composer), enabling one-click core updates through the Dashboard. To check for available updates, click **Check Now** from the site **Dashboard Dev** tab.
+
+### Drupal 9.2.11
+
+[Drupal 9.2.11](https://www.drupal.org/project/drupal/releases/9.2.11) is available and includes backwards-compatible improvements. This is a minor version (feature release) of Drupal 9 and is available for use on production sites. 
+
+### WordPress 5.9
+
+[WordPress 5.9](https://wordpress.org/news/2022/01/josephine/) is available for production sites and includes full site editing and the first default block theme. WordPress 5.9 comes installed with the Twenty Twenty-Two theme. 
+
+## New and Updated Features
+
+### Email Preferences in the New Dashboard
+
+You can now customize your personal settings in the New Dashboard to easily manage marketing communication from Pantheon.  
+
```

Comment:
@joa-pan I would consider changing this to something like: You can now customize the marketing communication you receive from Pantheon in the New Dashboard. From the Pantheon Dashboard got to > **Personal Settings**> **Notifications** > and click **Manage Preferences** under **Emails from Pantheon**.

But that might be too detailed??

Reply:
That's great! Thanks @whitneymeredith